src/spdx_tools/spdx3/model/build/build.py

```diff
@@ -0,0 +1,55 @@
+# SPDX-FileCopyrightText: 2023 spdx contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+from dataclasses import field
+from datetime import datetime
+from typing import Dict, List, Optional
+
+from spdx_tools.common.typing.dataclass_with_properties import dataclass_with_properties
+from spdx_tools.common.typing.type_checks import check_types_and_set_values
+from spdx_tools.spdx3.model import CreationInformation, Element, ExternalIdentifier, ExternalReference, IntegrityMethod
+
+
+@dataclass_with_properties
+class Build(Element):
+    build_type: str = None
+    build_id: Optional[str] = None
+    config_source_entrypoint: List[str] = field(default_factory=list)
+    config_source_uri: List[str] = field(default_factory=list)
+    config_source_digest: List[str] = field(default_factory=list)
+    parameters: Dict[str, str] = field(default_factory=dict)
+    build_start: Optional[datetime] = None
+    build_end: Optional[datetime] = None
+    environment: Dict[str, str] = field(default_factory=dict)
+
+    def __init__(
+        self,
+        spdx_id: str,
+        creation_info: CreationInformation,
+        build_type: str,
+        name: Optional[str] = None,
+        summary: Optional[str] = None,
+        description: Optional[str] = None,
+        comment: Optional[str] = None,
+        verified_using: List[IntegrityMethod] = None,
+        external_references: List[ExternalReference] = None,
+        external_identifier: List[ExternalIdentifier] = None,
+        extension: None = None,
+        build_id: Optional[str] = None,
+        config_source_entrypoint: List[str] = None,
+        config_source_uri: List[str] = None,
+        config_source_digest: List[str] = None,
```

Comment:
According to the md file his should be `List[Hash]`

Reply:
good catch! Fixed